omnibus/config/templates/installer/datadog-agent-sysprobe-exp.service.erb

```diff
@@ -1,7 +1,6 @@
 [Unit]
 Description=Datadog System Probe Experiment
 Requires=sys-kernel-debug.mount
-Before=datadog-agent.service
```

Comment:
For my understanding why are you not doing `Before=datadog-agent-exp.service` here ?

Reply:
Good question:
First about `After` - oneshot type does not support well as it waits for completion of the oneshot.
Since there's no guarantee that agents have a special ordering between starts (any service can restart at any point) I did not add After/Before on the exp units.

I missed it initially but all agents use After except sysprobe that uses Before https://github.com/DataDog/datadog-agent/pull/4883, I'll check why when I'm back